dataland-frontend/src/components/resources/frameworkDataSearch/lksg/LksgCompanyDataTable.vue

```diff
@@ -41,28 +41,28 @@
         :key="reportingPeriod.dataId"
```

Comment:
Why have we set `reorderableColumns` to `true` for the `<DataTable>`? 
Has UX/UI requested us to enable this functionality?

Reply:
It is indeed possible, I've tried it locally. But I don't think that we ever explicitily decided that we want this.
Therefore it is ok that you took it away and I close this conversation.